common/dex/base/dex-passwords.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
This file is not added to `kustomization.yaml` in `resources` section. Installing Kubeflow with this version renders dex as `CreateContainerConfigError`:
```
$ kubectl  -n auth describe pod -l app=dex
(...)
Events:
  Type     Reason     Age                From               Message
  ----     ------     ----               ----               -------
  Normal   Scheduled  39s                default-scheduler  Successfully assigned auth/dex-86f655645d-qc92d to kind-worker2
  Normal   Pulled     11s (x4 over 38s)  kubelet            Container image "ghcr.io/dexidp/dex:v2.36.0" already present on machine
  Warning  Failed     11s (x4 over 38s)  kubelet            Error: secret "dex-passwords" not found
```